common/src/main/java/com/wynntils/models/ingredients/type/IngredientTier.java

```diff
@@ -1,56 +0,0 @@
-/*
- * Copyright © Wynntils 2022.
- * This file is released under AGPLv3. See LICENSE for full license details.
- */
-package com.wynntils.models.ingredients.type;
-
-import com.google.gson.annotations.SerializedName;
-import com.wynntils.utils.colors.CustomColor;
-import net.minecraft.ChatFormatting;
-
-public enum IngredientTier {
```

Comment:
Both me and Poke mentioned this already, but why was this removed?

Reply:
I thought it mixed different areas of concern. How to represent the tier as a string was only really needed for the guide to know. The background color as well, and as I said above, my plan was to change so the guide item stack did not duplicate features with hard-coded values. But I never got around to fixing it. So, it might be that the best/simplest version is to restore this for now, and then I can see if I can make it superfluous in an upcoming PR.